apps/projects/serializers.py

```diff
@@ -209,6 +209,10 @@ class ModerationProjectSerializer(serializers.ModelSerializer):
     comment_count = serializers.SerializerMethodField()
     moderation_detail_url = serializers.SerializerMethodField()
 
+    def __init__(self, *args, **kwargs):
+        self.now = kwargs.pop("now")
+        super().__init__(args, kwargs)
+
```

Comment:
@goapunk I just added this to make test work, not sure how cool that is but as it's not used in mB project serializer I thought it was ok?